features/old/applications/providers/bulk_operations/send_emails.feature

```diff
@@ -76,3 +76,13 @@ Feature: Mass email bulk operations
     Given the email will fail when sent
     And I press "Send" and I confirm dialog box within colorbox
     Then I should see the bulk action failed with account "jane"
+
+Scenario: Emails can't be sent without subject
```

Comment:
Awesome, glad you were able to add a test for this.

Reply:
I added test but it keeps failing,  I added its path in the paths.rb file.
when I run the test it doesn't show any specific error but just failed.
I have added similar kind of test before which worked but not for this case.
Not sure if it works for files html.liquid also . If you have any idea about this feel free to share so I can check.